Assets/Scripts/DDAA/Variables/EnemySpawnDDAA.cs

```diff
@@ -34,8 +34,8 @@ public static EnemySpawnDDAA Instance
     // All of the parameters below are the ones to change, when adjusting the DDA (unless there's a bug)
 
     //static parameters
-    private static readonly float baseSpawnPoint = 4f;
-    public static readonly int minSpawnAmount = 7;
+    private static readonly float baseSpawnPoint = 0.5f;
```

Comment:
Why u change this? We also adjusted in our balancing stuff

Reply:
ah fuck left it from testing :DD